libraries/AP_HAL/Device.h

```diff
@@ -26,13 +26,24 @@
  */
 class AP_HAL::Device {
 public:
+    enum BusType {
+        I2C,
+        SPI,
```

Comment:
Nah... IMO This could continue the way it was (before applying this PR), since we can easily use `Device::I2C` and `Device::SPI`.